extensions/src/test/java/org/apache/falcon/extensions/store/ExtensionStoreTest.java

```diff
@@ -104,32 +104,57 @@ public void clean() {
         }
     }
 
-
     @Test
     public void testRegisterExtension() throws IOException, URISyntaxException, FalconException{
-        createLibs();
-        createReadmeAndJar();
-        createMETA();
+        String extensionPath = EXTENSION_PATH + "testRegister";
```

Comment:
Can you add code to clean up all the fs directories that are getting created at the end of all the tests. You can clean up the root directory EXTENSION_PATH directly.